extensions/table/table_entry.go

```diff
@@ -12,13 +14,15 @@ TableEntry represents an entry in a table test.  You generally use the `Entry` c
 type TableEntry struct {
 	Description string
 	Parameters  []interface{}
-	Pending     bool
-	Focused     bool
+	Flag        types.FlagType
+
+	// TODO: what if this isn't set?
```

Comment:
Good point. I don't have a clear answer. In #666 I just print the surrounding `DescribeTable` whenever it's not set.